VGC/frontend/src/pages/GamePage/GamePage.jsx

```diff
@@ -26,6 +26,28 @@ const GamePage = () => {
     fetchGame();
   }, [id]);
 
+  const handleAddToWishlist = async () => {
+    try {
+      const token = localStorage.getItem("token");
+      console.log("JWT token:", token);
+      const response = await fetch("http://localhost:5030/api/wishlist/", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+          "x-auth-token": token
+        },
+        body: JSON.stringify({ gameId: id })
+      });
+      if (!response.ok) {
+        throw new Error("Failed to add game to wishlist");
+      }
+      alert("Game added to wishlist successfully!");
+    } catch (error) {
+      console.error("Error adding game to wishlist:", error);
+      alert("Failed to add game to wishlist");
+    }
+  };
+
```

Comment:
for your add to wishlist, your using token to get the user I'm assuming. Im not sure but the x-auth-token is passed to req in api/wishlist correctly right?